README.md

```diff
@@ -1,7 +1,31 @@
-# Vue example app for Nextcloud
+# Nextcloud Photos
+**📸 Your memories under your control**
 
-1. ☁ Clone this into your `apps` folder of your Nextcloud
-2. 👩‍💻 In a terminal, run the command `make dev-setup` to install the dependencies
-3. 🏗 Then to build the Javascript whenever you make changes, run `make build-js`
-4. ✅ Enable the app through the app management of your Nextcloud
-5. 🎉 Partytime!
+![](screenshots/screenshot1.png)
+
+## Why is this so awesome?
+- **📸 Beautiful Photo und Video Timeline**
+- **⭐ Favorites and Tagging:** Favorite and tag your photos!
+- **Slideshow and easy sharing** Show slideshows and share your photos or albums easy
+- **Albums:** Create albums from your contents
+    
+    Future plans:
+- ...
+
+## 🚀 Installation  
+In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu. Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating the thumnails. 
```

Comment:
Should be `« Apps »`

Reply:
That’s only the french way. ;)

Let’s just go for:

> In your Nextcloud, go to the Apps management and enable the Photos app.

Reply:
Fine ^^